test/e2e/single_zone_e2e_test.go

```diff
@@ -43,19 +41,8 @@ var (
 	instance *remote.InstanceInfo
 	//gceCloud *gce.CloudProvider
 	nodeID string
-
-	project        = flag.String("project", "", "Project to run tests in")
-	serviceAccount = flag.String("service-account", "", "Service account to bring up instance with")
-	runInProw      = flag.Bool("run-in-prow", false, "If true, use a Boskos loaned project and special CI service accounts and ssh keys")
 )
 
-func TestE2E(t *testing.T) {
-	flag.Parse()
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "Google Compute Engine Persistent Disk Container Storage Interface Driver Tests")
-
-}
-
 var _ = BeforeSuite(func() {
```

Comment:
should the before/after suite also go in the common e2e?

Reply:
I don't think so. Because these before and after are specific to single zonal case. I envision another test file that has mutlizonal tests that will need its own separate setup and probably some refactoring :)